designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,263 @@
+- Repo: No ESLint repos, only the ones mentioned below (owned by private
+  persons)
+- Start Date: 2022-05-31
+- RFC PR:
+- Authors: [@aladdin-add](https://github.com/aladdin-add),
+  [@MichaelDeBoey](https://github.com/MichaelDeBoey),
+  [@ota-meshi](https://github.com/ota-meshi) &
+  [@voxpelli](https://github.com/voxpelli)
+
+# `@eslint-community` GitHub organisation
+
+## Summary
+
+<!-- One-paragraph explanation of the feature. -->
+
+Facilitate a GitHub organisation
+([@eslint-community](https://github.com/eslint-community)) where community
+members could maintain widely used ESLint related packages that would otherwise
+get abandoned.
+
+## Motivation
+
+<!-- Why are we doing this? What use cases does it support? What is the expected
+outcome? -->
+
+There a some widely used ESLint related packages that aren't actively maintained
+anymore and we'd like to transfer them to a community owned GitHub organization
+(`@eslint-community`), so we can give them back to the community.
+
+## Detailed Design
+
+<!--
+   This is the bulk of the RFC.
+
+   Explain the design with enough detail that someone familiar with ESLint
+   can implement it by reading this document. Please get into specifics
+   of your approach, corner cases, and examples of how the change will be
+   used. Be sure to define any new terms in this section.
+-->
+
+### How do you envision `@eslint-community`?
+
+Just like the [@jest-community](https://github.com/jest-community) org, the
```

Comment:
If we want to reference the `@jest-community`, then we should reference some kind of explanation of theirs on what their goal and intention is.

Reply:
I think it would be a good idea for  @G-Rath & @SimenB to give us some more insights into what the exact goal & intention of @jest-community is.

Reply:
> I think it would be a good idea for @G-Rath & @SimenB to give us some more insights into what the exact goal & intention of @jest-community is.

Did you ping them outside of this comment? Because I would imagine their GitHub inboxes to be a nightmare

Reply:
I'll send them a message on Discord/Twitter to ask if they want to give us the mentioned insights.

Reply:
I think @jeysal, @mattphillips, and @SimenB are the ones to ask here - while I'm a core maintainer of `eslint-plugin-jest`, I've not been involved in (or seen tbh) the @jest-community side of things.

I think that the goal of is it similar to what we're discussing here (or part of at least): a loose but centralized home for stuff `jest` core thinks is neat and worth having but doesn't belong in the core itself, and so a way to help ensure that these projects are not dependent on a one person's GitHub account in case they move on.

Reply:
Agreed.